examples/transactional-template-flow/main.bal

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/io;
+import ballerinax/mailchimp.'transactional as mailchimp;
+
+configurable string key = ?;
+const string serviceUrl = "https://mandrillapp.com/api/1.0";
```

Comment:
Do we need this URL here? It should have a default value (which is as same as this), right?
In case you need this as a constant, it should be `SERVICE_URL`.